controllers/v1/qa_model_controller_v1.py

```diff
@@ -14,7 +14,8 @@ def create_qa_model(request: dict) -> QAModelResponse:
     model = dto.to_model()
 
     try:
-        model = qa_model_service.create_qa_model(model)
+        #model = qa_model_service.create_qa_model(model)
```

Comment:
No need to comment out this line, it can just be removed

Reply:
Controller havent tested completely, thus I comment this out first

Reply:
Okay, that makes sense, worth making a note then that this should be updated after testing